src/components/JdsSimpleTable/SimpleTable.stories.js

```diff
@@ -0,0 +1,45 @@
+import JdsSimpleTable from './SimpleTable.vue'
+import { default as storybookMixin } from '../../utils/storybook'
+
+export default {
+  component: JdsSimpleTable,
+  title: 'Components/SimpleTable',
+}
+
+const Template = (args, context) => {
+  return {
+    name: 'JdsSimpleTableStories',
+    components: { JdsSimpleTable },
+    mixins: [storybookMixin(args, context)],
+    template: `
+    <jds-simple-table>
+      <thead>
+        <tr>
+          <th>Header Table</th>
+          <th>Header Table</th>
+          <th>Header Table</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Data Table</td>
+          <td>Data Table</td>
+          <td>Data Table</td>
+        </tr>
+        <tr>
+          <td>Data Table</td>
+          <td>Data Table</td>
+          <td>Data Table</td>
+        </tr>
+        <tr>
+          <td>Data Table</td>
+          <td>Data Table</td>
+          <td>Data Table</td>
+        </tr>
+      </tbody>
+    </jds-simple-table>
+    `,
+  }
+}
+
+export const Default = Template.bind({})
```

Comment:
Low Priority
It's better to be given a new line

Reply:
thankyou, will fix it asap

Reply:
Done [`da0ee94`](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/42/commits/da0ee94725188f21503b0f33f44ee08915c087bd)